tket/tests/test_Routing.cpp

```diff
@@ -1,2712 +0,0 @@
-    tket::Routing router(circ, test_arc);
-    std::pair<tket::Circuit, bool> outcirc = router.solve();
-    REQUIRE(outcirc.second == true);
-    CHECK(respects_connectivity_constraints(outcirc.first, test_arc, false));
-  }
-  GIVEN("A failing case, undirected") {
-    Circuit circ(3);
-    circ.add_op<unsigned>(OpType::CX, {0, 2});
-    reassign_boundary(circ);
-    REQUIRE_FALSE(respects_connectivity_constraints(circ, arc, false));
-  }
-  GIVEN("A working case, undirected") {
-    Circuit circ(3);
-    circ.add_op<unsigned>(OpType::CX, {0, 1});
-    reassign_boundary(circ);
-    REQUIRE(respects_connectivity_constraints(circ, arc, false));
-  }
-  GIVEN("A failing case, directed") {
-    Circuit circ(3);
-    circ.add_op<unsigned>(OpType::CX, {0, 1});
-    reassign_boundary(circ);
-    REQUIRE_FALSE(respects_connectivity_constraints(circ, arc, true));
-  }
-  GIVEN("A working case, directed") {
-    Circuit circ(3);
-    circ.add_op<unsigned>(OpType::CX, {1, 0});
-    reassign_boundary(circ);
-    REQUIRE(respects_connectivity_constraints(circ, arc, true));
-  }
-  GIVEN("A failing case, undirected, with SWAP") {
-    Circuit circ(3);
-    Vertex swap_v = circ.add_op<unsigned>(OpType::SWAP, {1, 2});
-
-    EdgeVec swap_outs = circ.get_all_out_edges(swap_v);
-    circ.dag[swap_outs[0]].ports.first = 1;
-    circ.dag[swap_outs[1]].ports.first = 0;
-
-    circ.add_op<unsigned>(OpType::CX, {0, 1});
-    reassign_boundary(circ);
-    REQUIRE_FALSE(respects_connectivity_constraints(circ, arc, false));
-  }
-  GIVEN("A working case, undirected, with SWAP") {
-    Circuit circ(3);
-    Vertex swap_v = circ.add_op<unsigned>(OpType::SWAP, {1, 2});
-
-    EdgeVec swap_outs = circ.get_all_out_edges(swap_v);
-    circ.dag[swap_outs[0]].ports.first = 1;
-    circ.dag[swap_outs[1]].ports.first = 0;
-
-    circ.add_op<unsigned>(OpType::CX, {0, 2});
-    reassign_boundary(circ);
-    REQUIRE(respects_connectivity_constraints(circ, arc, false));
-  }
-  GIVEN("A failing case, directed, with SWAP") {
-    Circuit circ(3);
-    Vertex swap_v = circ.add_op<unsigned>(OpType::SWAP, {1, 0});
-
-    EdgeVec swap_outs = circ.get_all_out_edges(swap_v);
-    circ.dag[swap_outs[0]].ports.first = 1;
-    circ.dag[swap_outs[1]].ports.first = 0;
-
-    circ.add_op<unsigned>(OpType::CX, {1, 0});
-    reassign_boundary(circ);
-    REQUIRE_FALSE(respects_connectivity_constraints(circ, arc, true));
-  }
-  GIVEN("A working case, directed, with SWAP") {
-    Circuit circ(3);
-    Vertex swap_v = circ.add_op<unsigned>(OpType::SWAP, {1, 0});
-
-    EdgeVec swap_outs = circ.get_all_out_edges(swap_v);
-    circ.dag[swap_outs[0]].ports.first = 1;
-    circ.dag[swap_outs[1]].ports.first = 0;
-
-    circ.add_op<unsigned>(OpType::CX, {0, 1});
-    reassign_boundary(circ);
-    REQUIRE(respects_connectivity_constraints(circ, arc, false));
-  }
-}
-
-SCENARIO("Test decompose_SWAP_to_CX pass", "[routing]") {
-  Architecture arc({{0, 1}, {1, 2}, {2, 3}, {3, 4}, {4, 0}});
```

Comment:
It is hard to check that all tests have been replicated under new names. As a quick test I did a search for this line, for example, and didn't find it. Can you confirm that we're not losing any test coverage?

Reply:
As with other comment, I hvae made changes and am now confident c++ tests cover and exceed old coverage.